projects/vendor/src/yotpo/baseyotpo/baseyotpo.service.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { BaseyotpoService } from './baseyotpo.service';
+
+describe('BaseyotpoService', () => {
```

Comment:
Please provide testing for the functions of the service.

Reply:
done